CMake/cdat_modules/pyspharm_pkg.cmake

```diff
@@ -1,10 +1,14 @@
 set(PYSPHARM_MAJOR 1)
 set(PYSPHARM_MINOR 0)
-set(PYSPHARM_PATCH 7)
+set(PYSPHARM_PATCH 8)
 set(PYSPHARM_VERSION ${PYSPHARM_MAJOR}.${PYSPHARM_MINOR}.${PYSPHARM_PATCH})
 set(PYSPHARM_URL ${LLNL_URL})
 set(PYSPHARM_GZ pyspharm-${PYSPHARM_VERSION}.tar.gz)
-set(PYSPHARM_MD5 e19897ec64d9e861ef6b93bf649a433c)
+set(PYSPHARM_MD5 7b3a33dd3cbeaa4b8bf67ed5bd210931)
 set(PYSPHARM_SOURCE ${PYSPHARM_URL}/${PYSPHARM_GZ})
 
-add_cdat_package_dependent(pyspharm "" "" OFF "CDAT_BUILD_LEAN" OFF)
+if (CDAT_BUILD_ALL)
```

Comment:
@sankhesh we should do that for rpy2, I noticed it doesn't build if BUILD_MODE=ALL